**1. Из колоды в 52 карты извлекаются случайным образом 4 карты.**

**a) Найти вероятность того, что все карты – крести.**

In [ ]:
from math import factorial

In [ ]:
def combination(n, k):
  return int(factorial(n)/ (factorial(k) * factorial(n-k)))

In [ ]:
C_4_13 = combination(13, 4)
C_4_13

715

In [ ]:
C_4_52 = combination(52, 4)
C_4_52

270725

In [ ]:
P = C_4_13/C_4_52
P

0.0026410564225690276

Ответ: P = 0.00264

**б) Найти вероятность, что среди 4-х карт окажется хотя бы один туз.**

In [ ]:
C_4_48 = combination(48, 4)
C_4_48

194580

In [ ]:
C = C_4_52 - C_4_48
C

76145

In [ ]:
P = C/C_4_52
P

0.2812632745405855

Ответ: P = 0,28126

**2. На входной двери подъезда установлен кодовый замок, содержащий десять кнопок с цифрами от 0 до 9. Код содержит три цифры, которые нужно нажать одновременно. Какова вероятность того, что человек, не знающий код, откроет дверь с первой попытки?**

In [ ]:
C_3_3=combination(3,3)

In [ ]:
C_3_10=combination(10,3)
C_3_10

120

In [ ]:
P = C_3_3 / C_3_10
P

0.008333333333333333

Ответ: P=0.008

**3. В ящике имеется 15 деталей, из которых 9 окрашены. Рабочий случайным образом извлекает 3 детали. Какова вероятность того, что все извлеченные детали окрашены?**

In [ ]:
C_3_9 = combination(9, 3)
C_3_9

84

In [ ]:
C_3_15 = combination(15, 3)
C_3_15

455

In [ ]:
P = C_3_9 / C_3_15
P

0.18461538461538463

Ответ: P = 0.1846

**4. В лотерее 100 билетов. Из них 2 выигрышных. Какова вероятность того, что 2 приобретенных билета окажутся выигрышными?**

In [ ]:
C_2_100 = combination(100, 2)
C_2_100

4950

In [ ]:
C_2_2 = combination(2, 2)

In [ ]:
P = C_2_2 / C_2_100
P

0.00020202020202020202

Ответ: P = 0.0002